Release 5.1.8 (#67)
neo4j/docs-kafka-connector: modules/ROOT/pages/changelog.adoc

a|
label:functionality[]
label:new[]

Support for `decimal`, `date`, `time` and `timestamp` Kafka Connect types in sink connector.
a|
When an incoming message contained one of the built-in Kafka Connect types (`decimal`, `date`, `time` and `timestamp`), the message could not be processed by the sink connector since theses types were not recognized by the sink connector.
Support for these types is now made available.

[NOTE]
====
To avoid a data loss, `decimal` typed values are converted to `string` values.
====

a|
label:bug[]
label:fixed[]

Rethrow exceptions if no errant record reporter is not set.
| When a sink connector is configured without any error handling options, errant record reporter is not set by Kafka Connect and exceptions raised were silently ignored.
These exceptions are no longer ignored, and re-thrown if no errant record reporter is set.

a|
label:bug[]
label:fixed[]

User provided Cypher queries that return results were failing in cypher sink strategy.
| When a sink connector is configured with cypher strategy and the provided query included `RETURN` clause, the internally re-written Cypher query was not being generated correctly and connector ended up failing.

> The Cypher generation logic is now fixed and these type of queries can also be specified.

Reviewer: ```suggestion
The Cypher generation logic is now fixed and these type of queries are now supported.
```
